website.js

```diff
@@ -357,23 +357,40 @@ app.get('/history', function(req, res) {
 
 app.get('/poll', function(req, res) {
   connection.query('select * from pollquestions ORDER BY id DESC LIMIT 1', function(err, result) {
-    res.redirect('/poll/' + result[0].id)
+    if(result[0].id != undefined) {
+      res.redirect('/poll/' + result[0].id)
+    } else {
+      res.render('error404.html')
+    }
   });
 });
 
 app.get('/poll/result', function(req, res) {
```

Comment:
Why does this route exist? If users want to view a random poll result, they can just go to `/poll/result`. It seems a bit strange 🤔